charts/consul/templates/server-service.yaml

```diff
@@ -39,6 +39,9 @@ spec:
       port: 8501
       targetPort: 8501
     {{- end }}
+    - name: grpc
+      port: 8503
+      targetPort: 8503
```

Comment:
since it's not a host port, could it be 8502?

Reply:
you mean the port is 8502 but the targetPort remains 8503?

Reply:
Because it will need to be a hostPort on kind. Then it will clash with the client GRPC port.

Reply:
you're totally right, I looked at the service instead of the stateful set!